<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Database Editor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="executeButton">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionCommit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionCreate_Table"/>
    <addaction name="actionAdd_Entity"/>
    <addaction name="actionAdd_Attribute"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Table"/>
    <addaction name="actionRemove_Entity"/>
    <addaction name="actionRemove_Attribute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
  </widget>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionCommit">
   <property name="text">
    <string>Commit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCreate_Table">
   <property name="text">
    <string>Create Table</string>
   </property>
  </action>
  <action name="actionAdd_Entity">
   <property name="text">
    <string>Add Entity</string>
   </property>
  </action>
  <action name="actionAdd_Attribute">
   <property name="text">
    <string>Add Attribute</string>
   </property>
  </action>
  <action name="actionRemove_Table">
   <property name="text">
    <string>Remove Table</string>
   </property>
  </action>
  <action name="actionRemove_Entity">
   <property name="text">
    <string>Remove Entity</string>
   </property>
  </action>
  <action name="actionRemove_Attribute">
   <property name="text">
    <string>Remove Attribute</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
